### PR TITLE
MM-31154 Add CI step to validate store mocks

### PR DESCRIPTION
**Pull request by reflog**
#### Summary

implemented validating mocks during ci
#### Ticket Link

https://mattermost.atlassian.net/browse/MM-31154
#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -128,7 +128,18 @@ jobs:
...
+      - image: mattermost/mattermost-build-server:20200731_golang-1.14.6
+    working_directory: ~/mattermost
+    steps:
+      - attach_workspace:
+          at: ~/mattermost/
+      - run:
+          command: |
+            cd mattermost-server
+            make einterfaces-mocks store-mocks
+            if [[ -n $(git status --porcelain) ]]; then echo "Please update the mocks using make make einterfaces-mocks store-mocks"; exit 1; fi
```
Comment (metanerd) [inline .circleci/config.yml:142]: > using make make einterfaces-mocks store-mocks

Is make make correct or just typo? If yes, maybe some single quotation could make it clearer?

Comment (metanerd) [reply .circleci/config.yml:142]: Looks good to me otherwise! Thank you so much!

**Review by metanerd — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (reflog) [reply .circleci/config.yml]: typo fixed!

**Comment by reflog** [comment]
@metanerd - nice. the CI caught an issue in master 🤦 
I'll find the responsible party to bet it fixed

**Comment by metanerd** [comment]
Yay, it works! :grinning:

**Comment by metanerd** [comment]
Sorry for the conflicts!

**Pull request merged**